opensearch/catalog-opensearch-source/src/main/java/ddf/catalog/source/opensearch/OpenSearchSource.java

```diff
@@ -273,10 +291,13 @@ public SourceResponse query(QueryRequest queryRequest) throws UnsupportedQueryEx
                 Metacard metacard = null;
                 List<Result> resultQueue = new ArrayList<Result>();
                 try {
-                    metacard = inputTransformer.transform(responseStream);
-                } catch (IOException e) {
-                    LOGGER.debug("Problem with transformation.", e);
-                } catch (CatalogTransformerException e) {
+                    final byte[] bytes = IOUtils.toByteArray(responseStream);
```

Comment:
@beyelerb That's a good point. I know @stustison linked to at least one other location. I think I'll go with the system property approach (with a reasonable default value). We might want to consider making a ticket to update other usages of that pattern to also reference the system property.